__tests__/plugin.test.js

```diff
@@ -16,6 +16,33 @@ describe('babel plugin', () => {
     process.env.REANIMATED_PLUGIN_TESTS = 'jest';
   });
 
+  it('injects its version', () => {
+    const input = `
+      function foo() {
+        '__Reanimated Babel Plugin version injection entry point';
```

Comment:
Honestly we don't need `__` in the beginning, also directives usually start with a verb, like `'use strict';`. Personally I like the following name more:
```suggestion
        'inject Reanimated Babel plugin version';
```